plugins/catalog-backend/src/service/router.ts

```diff
@@ -173,6 +173,11 @@ export async function createRouter(
         const output = await locationService.getLocation(id);
         res.status(200).json(output);
       })
+      .get('/locations/:type/refresh', async (req, res) => {
```

Comment:
This is the old catalog stuff here, so we shouldn't touch this. This will be removed in future versions.

Reply:
Ah, changes I saw must have been from before release, will remove.